server/column_scope.cc

```diff
@@ -145,6 +145,52 @@ static CellValue extractVal(const CellValue & val, CellValue *)
     return val;
 }
 
+namespace {
+    // Made to init and hold a non POD array of dynamic size in a memory safe
+    // way.
+    template<typename T, size_t MAX_STACK_ENTRIES=4096/sizeof(T)>
```

Comment:
I don't get why you divide by the size of the object here.  This controls the number of entries not the number of bytes, right?